src/main/java/uk/gov/justice/hmpps/prison/service/AdjustmentService.java

```diff
@@ -16,7 +16,7 @@ public class AdjustmentService {
     private final OffenderBookingRepository offenderBookingRepository;
 
     @Transactional(readOnly = true)
-    @VerifyBookingAccess(overrideRoles = {"SYSTEM_USER", "GLOBAL_SEARCH", "VIEW_PRISONER_DATA"})
```

Comment:
Only calculate-release-dates-client calls this endpoint which does not have SYSTEM_USER